tests/integration/pilot/common/routing.go

```diff
@@ -2775,6 +2775,89 @@ type vmCase struct {
 	host string
 }
 
+func ExternalServiceCases(t TrafficContext) {
+	type Port struct {
+		Name       string
+		Protocol   string
+		Number     int
+		Resolution string
+	}
+	makeSE := func(resolution string, ports ...Port) string {
+		return tmpl.MustEvaluate(`
+apiVersion: networking.istio.io/v1alpha3
+kind: ServiceEntry
+metadata:
+  name: external-google
+spec:
+  hosts:
+  - "www.google.com"
```

Comment:
we do not want to depend on external sites, see https://github.com/istio/istio/commit/cde394ec6832303cf1e3b67ce6435d6b89fe0bdc for an example of using a fake external one

Reply:
Ack; edited